pkg/controllers/disruption/emptiness.go

```diff
@@ -32,11 +31,13 @@ import (
 // Emptiness is a subreconciler that deletes empty candidates.
 type Emptiness struct {
 	consolidation
+	Validator
```

Comment:
nit: can we encapsulate the validator in the consolidation struct and inject the correct validation through the make consolidation call?

Reply:
We would have to make Consolidation exportable in order to access the Validator so that we can override it in testing. Because nothing else about consolidation is changing, I'd prefer to leave it as is.